test/functional/framework.go

```diff
@@ -176,8 +176,7 @@ func (f *FluentdFunctionalFramework) addOutputContainers(b *runtime.PodBuilder,
 	return nil
 }
 
-func (f *FluentdFunctionalFramework) WritesApplicationLogs(numOfLogs int) error {
-	msg := "2020-11-04T18:13:59.061892999+00:00 stdout F Functional test message $n"
```

Comment:
We may need an alternate function.  I think there is value in:
* writing just N number of messages regardless of format
* writing an explicit message so you can verify an explicit output
* taking a message and having the generator maybe add the current time, whether it is split, or not, etc

Reply:
Fixed